massgov/mayflower
# all/DP-27164-fix-google-translate-styling-caused-by-google

Fixed:
  - project: Assets
    component: GoogleTranslate, UtilityNav
    description: Fix Google Translate styles caused by a classname change by Google. (#1740)
    issue: DP-27164
    impact: Patch


To Test:
- please confirm that both implementations are fixed - including react and patternlab
- test in openmass https://github.com/massgov/openmass/pull/1871
